doc/data/messages/d/differing-param-doc/pylintrc

```diff
@@ -0,0 +1,2 @@
+[main]
+load-extension: pylint.extensions.docparams
```

Comment:
This is the incorrect format. Please look at how other examples do this.

In this case this should work:
```suggestion
load-extension = pylint.extensions.docparams
```

Reply:
Still showing errors